Gemfile

```diff
@@ -18,14 +18,14 @@ allow_local = ENV['FAVOR_LOCAL_GEMS']
 # elsif allow_local
 #   gem 'openstudio-extension', github: 'NREL/openstudio-extension-gem', branch: 'develop'
 # end
-
+gem 'openstudio-extension', path: '../OpenStudio-extension-gem'
```

Comment:
@kflemin is this a temporary fix

Reply:
that shouldn't be like that, I'll fix it